x-pack/plugins/alerting/server/routes/maintenance_window/apis/find/find_maintenance_windows_route.ts

```diff
@@ -34,18 +58,16 @@ export const findMaintenanceWindowsRoute = (
       verifyAccessAndContext(licenseState, async function (context, req, res) {
         licenseState.ensureLicenseForMaintenanceWindow();
 
+        const query: FindMaintenanceWindowsRequestQueryV1 = req.query || {};
         const maintenanceWindowClient = (await context.alerting).getMaintenanceWindowClient();
 
-        const result: FindMaintenanceWindowsResult = await maintenanceWindowClient.find();
+        const options = transformFindMaintenanceWindowParamsV1(query);
+        const findResult: FindMaintenanceWindowsResult = await maintenanceWindowClient.find(
+          options
+        );
 
-        const response: FindMaintenanceWindowsResponseV1 = {
-          body: {
-            data: result.data.map((maintenanceWindow) =>
-              transformMaintenanceWindowToResponseV1(maintenanceWindow)
-            ),
-            total: result.data.length,
-          },
-        };
+        const response: FindMaintenanceWindowsResponseV1 =
+          transformFindMaintenanceWindowResponseV1(findResult);
```

Comment:
I think is better if the `body` is not part of the `FindMaintenanceWindowsResponseV1`. The reason is that the response should not encapsulate the implementation details of the router and contain only the schema of the MW response. This way if in the future the router changes its structure we will not have to change our schemas.